backend/src/models/reports/Tab_27_rpt_PA_Ministry.js

```diff
@@ -0,0 +1,16 @@
+//const dbConnection = require("@database/databaseConnection");
+//const { knex } = dbConnection();
+
+const getReport = (portfolio) => {
+  const query = "removeme"; //knex().raw();
+  return query;
+};
+
+module.exports = {
+  required: ["portfolio"],
```

Comment:
nice idea, front-loading the 'required' property in the model itself. it makes sense to keep it here instead in some JSON file on the front-end (separation of concerns)